src/v/model/tests/randoms.h

```diff
@@ -0,0 +1,73 @@
+
+/*
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#include "model/compression.h"
+#include "model/fundamental.h"
+#include "model/metadata.h"
+#include "model/timestamp.h"
+#include "test_utils/randoms.h"
+
+namespace model {
+inline model::topic_namespace random_topic_namespace() {
+    return {
+      tests::random_named_string<model::ns>(),
```

Comment:
nit

Reply:
it is a matter of taste i made it like this because 
```c++
tests::random_named_string<model::ns>();
``` 
is less verbose comparing to:
```c++
model::ns(random_generators::gen_alphanumeric(10));
```